configure.ac

```diff
@@ -734,6 +734,7 @@ if test x$TARGET_OS != xwindows; then
   AX_CHECK_COMPILE_FLAG([-fPIC],[PIC_FLAGS="-fPIC"])
 fi
 
+AX_CHECK_COMPILE_FLAG([-fstack-reuse=none],[HARDENED_CXXFLAGS="$HARDENED_CXXFLAGS -fstack-reuse=none"])
```

Comment:
Should have a comment here linking to the bug, because otherwise it's not obvious why you'd want this flag.

Reply:
Done